vignettes/p2visualisation.Rmd

```diff
@@ -0,0 +1,232 @@
+---
+title: "2. Visualisation"
+author: "James Hollway"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{2. Visualisation}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+
+
+# Why we graph
+
+Network visualisation is important and non-trivial.
+As Tufte (1983: 9) said:
+
+> "At their best, graphics are instruments for reasoning about quantitative information. Often the most effective way to describe, explore, and summarize a set of numbers – even a very large set – is to look at pictures of those numbers"
+
+As Brandes et al (1999) argue, visualising networks demands thinking about:
+
+- _substance_: a concise and precise delivery of insights to the researcher and/or readers
+- _design_: the ergonomics of function are 98% of the purpose of good design, aesthetics only 2%
+- and _algorithm_: the features of the e.g. the layout algorithm
+
+While there may be many dead-ends to exploratory visualisation,
+it is worth taking the time to make sure that your main points are
+easy to appreciate.
+
+On [her excellent and helpful website](https://kateto.net/network-visualization), 
+Katya Ognyanova outlines some key dimensions of control
+that network researchers have to play with:
+
+- vertex position (layout)
+- vertex shape
+- vertex color
+- vertex size
+- vertex labels
+- edge color
+- edge size
+- edge shape
+- edge arrows
+
+# Approaches to visualising networks in R
+
+There are a host of packages for plotting in R, 
+and for plotting networks in R.
+Plotting in R is typically based around two main approaches:
+the 'base' approach in R by default,
+and the 'grid' approach made popular by the famous and very flexible `{ggplot2}` package.^['gg' stands for the Grammar of Graphics.]
+Approaches to plotting _graphs_ or _networks_ in R can be similarly divided.
+
+The two classic packages are `{igraph}` and `{sna}`, both building upon the base R graphics engine.
+Newer packages [`{ggnetwork}`](https://www.r-bloggers.com/2016/03/ggnetwork-network-geometries-for-ggplot2/) and 
+[`{ggraph}`](https://ggraph.data-imaginist.com/index.html) build upon a grid approach.^[
+Others include: 'Networkly' for creating 2-D and 3-D interactive networks
+that can be rendered with plotly and can be easily integrated into
+shiny apps or markdown documents;
+'visNetwork' interacts with javascript (vis.js) to make interactive networks
+(http://datastorm-open.github.io/visNetwork/); and
+'networkD3' interacts with javascript (D3) to make interactive networks
+(https://www.r-bloggers.com/2016/10/network-visualization-part-6-d3-and-r-networkd3/).
+]
```

Comment:
Suggestion: Moves lines 56-63 to a footnote

Reply:
Lines 56-63 are a footnote.